TGM/src/main/java/network/warzone/tgm/modules/SpectatorModule.java

```diff
@@ -359,9 +359,9 @@ public void onQuit(PlayerQuitEvent event) {
     }
     
     @EventHandler
-    public void itemFrameItemRemoval(EntityDamageByEntityEvent e) {
-        if (e.getEntity() instanceof ItemFrame) {
-            e.setCancelled(true);
+    public void itemFrameItemRemoval(EntityDamageByEntityEvent event) {
```

Comment:
Should probably make use of `isSpectating(player)` in this method?

Reply:
indeed Aerh

Reply:
Fixed.